src/Tribe/Main.php

```diff
@@ -823,6 +826,9 @@ protected function addHooks() {
 			// Upgrade material.
 			add_action( 'init', array( $this, 'run_updates' ), 0, 0 );
 
+			// Update Capabilities
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 3d6c8473edecccac81690cc4dbf8e892</sub>